### PR TITLE
Change styles checkbox

### PR DESCRIPTION
**Pull request by AlexNti**
if this PR  applied it will fix the following bug [https://github.com/netdata/netdata-cloud/issues/233](url).
Also we made a small refactor in order to fix that bug.

### CODE REVIEW

File: .storybook/config.tsx [hunk 1]
```diff
@@ -12,6 +12,7 @@ import { ThemeProvider } from "styled-components"
 import Flex from "../src/components/templates/flex"
```
Comment (AlexNti) [inline .storybook/config.tsx:12]: The change at this file are irrelevant with the this issue. 
This change fixes a bug at storybook where labels does not change colour on dark theme.

File: .storybook/config.tsx [hunk 2]
```diff
@@ -58,10 +59,10 @@ addDecorator(story => {
               checked={isDarkTheme}
               colored={false}
               disabled={false}
-              Label={"asdas"}
+              Label={Text}
```
Comment (burbuli8ra) [inline .storybook/config.tsx:62]: Why is label with capital?

File: package.json [hunk 3]
```diff
@@ -34,6 +34,7 @@
     "@elastic/search-ui-site-search-connector": "^1.5.1",
     "@netdata/react-filter-box": "^1.0.0",
     "@prismicio/client": "^5.1.0",
+    "@testing-library/user-event": "^13.5.0",
```
Comment (burbuli8ra) [inline package.json:37]: 🎉

**Review by burbuli8ra — APPROVED** [review]

File: .storybook/config.tsx (hunk 2 above)
Comment (AlexNti) [reply .storybook/config.tsx:62]: hmmm, i believe this is capital to indicate that this input property expects a component. Check `toggle.js `for more information.

**Pull request merged**